src/modules/toDoList.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable linebreak-style */
+import { taskCompleted, taskEdit, updateValue } from './events.js';
+
+class ToDoList {
+  constructor() {
+    this.taskList = [];
+  }
+
+  createTask(description) {
+    const taskContainer = document.querySelector('.list');
+    const index = taskContainer.children.length + 1;
+    const li = document.createElement('li');
+    li.className = 'task';
+    const checkBox = document.createElement('input');
+    checkBox.type = 'checkbox';
```

Comment:
It would look cleaner if you used empty lines between assignments and use comments to make it easy to understand for other programmers